exchanges/kucoin/kucoin_test.go

```diff
@@ -41,45 +39,27 @@ const (
 	canManipulateRealOrders = false
 )
 
-var (
-	ku                                                        = &Kucoin{}
-	spotTradablePair, marginTradablePair, futuresTradablePair currency.Pair
-)
+var ku *Kucoin
+var spotTradablePair, marginTradablePair, futuresTradablePair currency.Pair
 
 func TestMain(m *testing.M) {
-	ku.SetDefaults()
-	cfg := config.GetConfig()
-	err := cfg.LoadConfig("../../testdata/configtest.json", true)
-	if err != nil {
+	ku = new(Kucoin)
+	if err := testexch.TestInstance(ku); err != nil {
 		log.Fatal(err)
 	}
 
-	exchCfg, err := cfg.GetExchangeConfig("Kucoin")
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	exchCfg.API.AuthenticatedSupport = true
-	exchCfg.API.AuthenticatedWebsocketSupport = true
-
-	exchCfg.API.Credentials.Key = apiKey
-	exchCfg.API.Credentials.Secret = apiSecret
-	exchCfg.API.Credentials.ClientID = passPhrase
 	if apiKey != "" && apiSecret != "" && passPhrase != "" {
+		ku.API.AuthenticatedSupport = true
+		ku.API.AuthenticatedWebsocketSupport = true
+		ku.API.CredentialsValidator.RequiresBase64DecodeSecret = false
```

Comment:
does this need to stay?

Reply:
I think so. Because before the Secret was being injected directly.
Now we're not injecting it, and the implication is that the user would put a plain secret in the top of the file, and we don't want it to be Base64Decoded.
Could change configtest instead maybe.

Do you suspect it of being wrong?